README.md

```diff
@@ -1,4 +1,98 @@
-# GetIt
+
+<div align="center">
+
+# 🛒 GETIT
+</div>
+    
+Streamlined college e-commerce: Simplifying merch buying and selling for students.
+
+
+## 🌟INTRODUCTION
+
+
+
+
+Welcome to our e-commerce website, your one-stop destination for exclusive merchandise tailored specifically for students of **PDPM IIITDMJ**. We are thrilled to bring you a wide range of products that showcase your college pride and allow you to represent your alma mater in style. Whether you're a freshman just starting your journey or a seasoned senior ready to graduate, our collection has something for everyone. From trendy apparel and accessories to unique gifts and study essentials, we've carefully curated a selection that captures the spirit of **PDPM IIITDMJ**. With high-quality materials, attention to detail, and designs that resonate with student life, our merchandise is sure to become your favorite go-to for all things college-related. Browse through our virtual aisles, shop with ease, and join us in celebrating your college experience with pride and passion.
+
+
+
+
+
+
+## ✨FEATURES
+
+- Clubs can sell their merch on the platform
+- Buy the merch you like 
+- Automatic payments for sellers no need for manual verification
+- Seller reports
+
+
+## 💻DEVELOPED WITH
+
+* [Visual Studio Code](https://code.visualstudio.com/) - A source code editor developed by Microsoft for Windows, Linux and macOS. It includes support for debugging, embedded Git control, syntax highlighting, intelligent code completion, snippets, and code refactoring
+
+
+---
+
+
+## 🔰 GETTING STARTED
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### PREREQUISITS
```

Comment:
Let's remove the whole `prerequisites` section

Reply:
Let's keep them since it avoids conflicts of different version and will have the contributor, have an idea of on which version the setup was done. So the person doesnt run into issues due to compatibility